### PR TITLE
Auth pages

### PR DESCRIPTION
**Pull request by Ash-D23**
Added auth pages
Added auth context and hooks
Modified navbar based on auth

Screenshot 2022-03-22 123639

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,8 +14,13 @@ import "./App.css";
 import Mockman from "mockman-js";
 import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
+import { useAuthContext } from "./Context/AuthContext/AuthContext";
+import RequireAuth from "./hooks/RequireAuth";
 
 function App() {
+
+  const { user } = useAuthContext()
+  console.log(user)
```
Comment (Pratik1005) [inline src/App.js:23]: remove console log

File: src/Context/AuthContext/AuthContext.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    </AuthContext.Provider>
+}
+
+// hook
+
+const useAuth = () => {
+    const [user, setuser] = useLocalStorage("user", null)
+
+    const signin = async (data) => {
+        console.log(data)
```
Comment (Pratik1005) [inline src/Context/AuthContext/AuthContext.js:24]: remove console log

**Review by Pratik1005 — APPROVED** [review]
Just remove the console logs , rest looks good

File: src/App.js (hunk 1 above)
Comment (Ash-D23) [reply src/App.js:23]: sure i will remove it i still need it to check cart management and wishlist management hence kept it

**Review by shobit1337 — APPROVED** [review]
Your code looks great. Good work @Ash-D23

**Pull request merged**